aiida_tbextraction/fp_run/wannier_input/_qe.py

```diff
@@ -13,7 +13,7 @@
 from aiida_tools import check_workchain_step
 from aiida_wannier90.calculations import Wannier90Calculation
 from aiida_quantumespresso.workflows.pw.base import PwBaseWorkChain
-from aiida_quantumespresso.calculations.pw2wannier90 import Pw2wannier90Calculation
+from aiida_tbextraction.fp_run.wannier_input.pw2wann import SplitPw2wannier90
```

Comment:
For consistency, we should name the file `pw2wannier` (sorry, didn't spot that one last time around).